redeem/gcodes/M2x.py

```diff
@@ -249,15 +251,28 @@ def execute(self, g):
             return
 
         fn = text.strip()
+        list_location = None
 
         if fn.startswith('/usb'):
             fn = fn.replace('/usb', USB_MOUNT_LOCATION)
+        list_location = USB_MOUNT_LOCATION
 
         if fn.startswith('/sd'):
             fn = fn.replace('/sd', SD_MOUNT_LOCATION)
+        list_location = SD_MOUNT_LOCATION
 
         if fn.startswith('/lcl'):
             fn = fn.replace('/lcl', LCL_MOUNT_LOCATION)
+        list_location = LCL_MOUNT_LOCATION
+
+    filemap = dict()
```

Comment:
I think this has the wrong level of indentation and should be part of the `execute` method for M23.

Reply:
I was going to ask you about that. I think I screwed it up when fixing the tabs/spaces mix.

Reply:
@ajmirsky what about the list_location indent? is that correct, or is it part of the if?

Reply:
`list_location` should be part of the `if` statement above it.